package.json

```diff
@@ -7,6 +7,7 @@
     "build": "jison src/gql.y src/gql.l -o dist/parser.js",
     "lint": "eslint lib test/*_spec.js",
     "test": "yarn build && yarn lint && NODE_ENV=testing mocha -- $(find test -name '*_spec.js')",
```

Comment:
does the `yarn lint` need to be in the test if it's executed in `posttest`?

Reply:
OH! Good catch! 🙈